atlascope/core/management/commands/populate.py

```diff
@@ -2,44 +2,34 @@
 import os
 from pathlib import Path
 
-from django.contrib.auth.models import User
 from django.contrib.gis.db.models import PolygonField
 from django.contrib.gis.geos.polygon import Polygon
 import djclick as click
-from guardian.shortcuts import assign_perm
-from oauth2_provider.models import Application
 from rest_framework.serializers import ValidationError
 
 from atlascope.core.models import Dataset, DatasetEmbedding, Investigation, Job
 
 POPULATE_DIR = 'atlascope/core/management/populate/'
 
 MODEL_JSON_MAPPING = [
-    (User, 'users.json'),
     (Dataset, 'datasets.json'),
     (Investigation, 'investigations.json'),
     (DatasetEmbedding, 'embeddings.json'),
     (Job, 'jobs.json'),
 ]
 
-DEFAULT_PASSWORD = 'letmein'
-
 DEFAULT_CLIENT_URI = 'http://localhost:8080/'
```

Comment:
we also don't need this

Reply:
See b40c3f7ff058b2170bc7beb669320f731b72c3b8.